<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VRConfWizardClass</class>
 <widget class="QMainWindow" name="VRConfWizardClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>COVISE VR Configuration Wizard[*]</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="centralWidgetVLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="Line" name="topScrollAreaLine">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1100</width>
         <height>678</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="scrollAreaVLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QStackedWidget" name="mainStackedWidget">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="page1">
           <layout class="QVBoxLayout" name="page1VLayout"/>
          </widget>
          <widget class="QWidget" name="page2">
           <layout class="QVBoxLayout" name="page2VLayout"/>
          </widget>
          <widget class="QWidget" name="page3">
           <layout class="QVBoxLayout" name="page3VLayout"/>
          </widget>
          <widget class="QWidget" name="page4">
           <layout class="QVBoxLayout" name="page4VLayout"/>
          </widget>
          <widget class="QWidget" name="page5">
           <layout class="QVBoxLayout" name="page5VLayout"/>
          </widget>
          <widget class="QWidget" name="page6">
           <layout class="QVBoxLayout" name="page6VLayout"/>
          </widget>
          <widget class="QWidget" name="page7">
           <layout class="QVBoxLayout" name="page7VLayout"/>
          </widget>
          <widget class="QWidget" name="page8">
           <layout class="QVBoxLayout" name="page8VLayout"/>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="Line" name="topNavigationLine">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="navigationWidget" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>60</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="navigationWidgetHLayout">
       <item>
        <widget class="QWidget" name="expertModeWidget" native="true">
         <layout class="QHBoxLayout" name="expertModeWidgetHLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <spacer name="expertModeHSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QCheckBox" name="expertModeCheckBox">
            <property name="text">
             <string>Expert mode</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="bottomHSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>404</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="buttonHLayout">
         <property name="spacing">
          <number>10</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="backnextHLayout">
           <property name="spacing">
            <number>1</number>
           </property>
           <item>
            <widget class="QPushButton" name="backButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>&lt; Back</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="nextButton">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Next &gt;</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="finishButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Finish</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="abortButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Abort</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="Line" name="bottomNavigationLine">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1100</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Exit">
    <property name="title">
     <string>E&amp;xit</string>
    </property>
    <addaction name="action_Finish"/>
    <addaction name="action_Abort"/>
   </widget>
   <widget class="QMenu" name="menu_Navigation">
    <property name="title">
     <string>&amp;Navigation</string>
    </property>
    <addaction name="action_Next"/>
    <addaction name="action_Back"/>
   </widget>
   <addaction name="menu_Exit"/>
   <addaction name="menu_Navigation"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Next">
   <property name="text">
    <string>&amp;Next</string>
   </property>
   <property name="shortcut">
    <string>Alt+N</string>
   </property>
  </action>
  <action name="action_Back">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Back</string>
   </property>
   <property name="shortcut">
    <string>Alt+B</string>
   </property>
  </action>
  <action name="action_Finish">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Finish</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_Abort">
   <property name="text">
    <string>&amp;Abort</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>abortButton</sender>
   <signal>clicked()</signal>
   <receiver>VRConfWizardClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>839</x>
     <y>658</y>
    </hint>
    <hint type="destinationlabel">
     <x>613</x>
     <y>389</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>finishButton</sender>
   <signal>clicked()</signal>
   <receiver>VRConfWizardClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>754</x>
     <y>658</y>
    </hint>
    <hint type="destinationlabel">
     <x>613</x>
     <y>373</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nextButton</sender>
   <signal>clicked()</signal>
   <receiver>VRConfWizardClass</receiver>
   <slot>next()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>668</x>
     <y>658</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>370</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>backButton</sender>
   <signal>clicked()</signal>
   <receiver>VRConfWizardClass</receiver>
   <slot>back()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>592</x>
     <y>658</y>
    </hint>
    <hint type="destinationlabel">
     <x>-2</x>
     <y>401</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Abort</sender>
   <signal>triggered()</signal>
   <receiver>VRConfWizardClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Finish</sender>
   <signal>triggered()</signal>
   <receiver>VRConfWizardClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Next</sender>
   <signal>triggered()</signal>
   <receiver>VRConfWizardClass</receiver>
   <slot>next()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Back</sender>
   <signal>triggered()</signal>
   <receiver>VRConfWizardClass</receiver>
   <slot>back()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>expertModeCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>VRConfWizardClass</receiver>
   <slot>emitExpertModeChanged(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>641</y>
    </hint>
    <hint type="destinationlabel">
     <x>-7</x>
     <y>641</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>back()</slot>
  <slot>next()</slot>
  <slot>emitExpertModeChanged(bool)</slot>
 </slots>
</ui>
